packages/api/lib/rulesHelpers.js

```diff
@@ -14,15 +14,21 @@ const { sqsQueueExists } = require('@cumulus/aws-client/SQS');
 const { invoke } = require('@cumulus/aws-client/Lambda');
 const { RulePgModel } = require('@cumulus/db');
 const { ValidationError } = require('@cumulus/errors');
+const { getRequiredEnvVar } = require('@cumulus/common/env.js');
 
 const { listRules } = require('@cumulus/api-client/rules');
 const { removeNilProperties } = require('@cumulus/common/util');
 
 const { handleScheduleEvent } = require('../lambdas/sf-scheduler');
 const { isResourceNotFoundException, ResourceNotFoundError } = require('./errors');
-const Rule = require('../models/rules');
 
 const log = new Logger({ sender: '@cumulus/rulesHelpers' });
+
+/**
```

Comment:
This should be moved up above the `const log ...` statement.

Reply:
Ah yup.

https://github.com/nasa/cumulus/pull/3149/commits/ebee8226b6ce6305610ea5e568f4de77d56d69ed